app/src/main/java/cm/aptoide/pt/UninstallObserver.kt

```diff
@@ -0,0 +1,14 @@
+package cm.aptoide.pt
+
+import cm.aptoide.pt.feature_oos.repository.UninstallObserver
+import kotlinx.coroutines.flow.Flow
+
+class UninstallObserver: UninstallObserver {
```

Comment:
Might be better to change the class name to UninstallObserverImpl or something similar